tsconfig.json

```diff
@@ -8,11 +8,15 @@
       "vitest/importMeta"
     ],
     "target": "es6",
+    "moduleResolution": "nodenext",
+    "module": "NodeNext"
```

Comment:
Yeah, should this be looked into? The error we're getting from GH actions.

Reply:
On it. Thought that I solved it with the one commit. But we still get a TS warning here in GH.

Reply:
Ok. So there is a difference in what my local linter and GH's linter is reporting here. Furthermore: The changes in this PR is not introducing anything that should cause this problem.
I suggest that instead of me headlessly trying to solve the problem, we should do it in a controlled manner. And therefor I will ignore the warnings for now.